### PR TITLE
[Feature] Add option to output boolean vector denoting if classes are present

### LINKED ISSUE #85: [Feature] Add option to output classes present when exporting
# Prerequisites

Please answer the following question for yourself before submitting an issue.

- [x] I checked to make sure that this feature has not been requested already.

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/tree/master/official/vision/beta

## 2. Describe the feature you request

Move postprocessing from raven to model. (incurs ~10ms) which is ~10-12% of inference time.

## 3. Additional context

Add any other context about the feature request here.

## 4. Are you willing to contribute it? (Yes or No)
Yes

### PR DESCRIPTION
**Pull request by rehohoho**
## Description
What feature/issue was addressed?
Add option to output boolean vector denoting if classes are present.
See relevant issue for motivation.

image
3rd from bottom is the boolean vector output

How was it resolved/added?
Added parameter on export definition for multihead, link it to running scripts.

Any dependencies required for the change?

## Issue reference

Please reference the issue this PR will close: #85.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (Specify)

## Tests

Any reproducable method of testing to verify changes? \
List relevant details for the test configuration.

## Checklist
- [x] I have performed a self-review of my own code. Feel free to destroy my PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by rehohoho** [comment]
Btw #81 implementation can be very similar to this. Define an extra output that uses a softmax on output logits when serving.

**Review by MelvinFMQ — APPROVED** [review]
lgtm, so this can potentially reduce 10% inference time?

**Comment by rehohoho** [comment]
I think not that much. maybe 5%?
Think the ops for this is not as efficient on GPU.
